src/kvstore/RocksEngine.cpp

```diff
@@ -340,6 +357,12 @@ int32_t RocksEngine::totalPartsNum() {
 
 ResultCode RocksEngine::ingest(const std::vector<std::string>& files) {
     rocksdb::IngestExternalFileOptions options;
+    options.move_files = true;
```

Comment:
Nice!

Reply:
If you set `verify_checksums_before_ingest`,  would you set `verify_checksums_readahead_size` to improve the performance?

Reply:
Yes